editor/package.json

```diff
@@ -12,11 +12,13 @@
   },
   "scripts": {
     "clean": "rm -rf lib/",
-    "test": "tsc && npm run --silent clean && jest --config ./jest.config.js",
-    "test-ci": "tsc && NODE_OPTIONS=--max_old_space_size=6656 xvfb-run -a npx jest --config ./jest.config.js --maxWorkers=1",
-    "test-debug": "tsc && npm run --silent clean && node --inspect-brk node_modules/.bin/jest --config ./jest.config.js --runInBand",
-    "test-watch": "tsc && npm run --silent clean && jest --config ./jest.config.js --watch",
-    "test-update-snapshot": "tsc && npm run --silent clean && jest --config ./jest.config.js --updateSnapshot",
+    "lint": "npx eslint \"src/**/*.{js,ts,tsx}\"",
+    "test": "tsc --noEmit && npm run --silent clean && npm run jest && npm run lint",
```

Comment:
Oooooh, if we include the `--noemit` does that mean we can bin off the `clean` step?

Reply:
I think so, but at this point we had the clean step in for so long, I'm terrified of removing it

Reply:
The clean step was just there to bin off the result of the `tsc` step. I _think_ however it would make more sense to have the clean step before it, since most of us run `tsc` separately during dev anyway, or just not include it :shrug: